packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -656,8 +656,8 @@ export const prodParasPolkadotCommon: EndpointOption[] = [
       'IBP-GeoDNS1': 'wss://sys.ibp.network/statemint',
       'IBP-GeoDNS2': 'wss://sys.dotters.network/statemint',
       OnFinality: 'wss://statemint.api.onfinality.io/public-ws',
-      Parity: 'wss://statemint-rpc.polkadot.io'
-      // RadiumBlock: 'wss://statemint.public.curie.radiumblock.co/ws' // https://github.com/polkadot-js/apps/issues/9240
+      Parity: 'wss://statemint-rpc.polkadot.io',
+      RadiumBlock: 'wss://statemint.public.curie.radiumblock.co/ws' 
```

Comment:
```suggestion
      RadiumBlock: 'wss://statemint.public.curie.radiumblock.co/ws'
```

This should fix the CI linting issues